pytest_selenium/pytest_selenium.py

```diff
@@ -93,13 +97,27 @@ def capabilities(request, driver_class, chrome_options, firefox_options,
         if all([key, options]):
             capabilities[key] = _merge(
                 capabilities.get(key, {}), options.get(key, {}))
-    capabilities_marker = request.node.get_marker('capabilities')
-    if capabilities_marker is not None:
+    capabilities_marker = _get_marker(request.node, 'capabilities')
+    LOGGER.debug('Capabilities from marker: {}'.format(capabilities_marker))
+    if capabilities_marker:
         # add capabilities from the marker
-        capabilities.update(capabilities_marker.kwargs)
+        capabilities.update(capabilities_marker)
     return capabilities
 
 
+def _get_marker(node, name):
+    # get_marker is deprecated
+    # https://docs.pytest.org/en/latest/mark.html#marker-revamp-and-iteration
+    try:
+        markers = dict()
+        for each in node.iter_markers(name):
+            markers.update(each.kwargs)
```

Comment:
As I understand it, this will find the `capabilities` markers applied to the various scopes and methods in use. The capabilities from each marker will be updated by the next. Whilst this it probably what we should have been doing all along, it could cause some confusion and potential breakage to existing users. I would suggest adding debug level logging for each marker found with as much context as we can. That way, a user can turn on debug logging and understand where/why any capabilities are being overwritten.

Reply:
I did some testing with module and function level markers, and that code behaves just as `get_markers()` did. That is, if there's a conflict between module and function, the function wins.

With that said, I can see if I can add more debugging.

Reply:
Oh so we don't iterate through them? That's interesting.. then what are we iterating through? Multiple markers with the same name on a single object?